src/olmo_core/internal/model_ladder.py

```diff
@@ -142,16 +145,17 @@ def main(ladder_builder: Callable[[str], ModelLadder]):
 [b magenta]dry_run:[/]     Pretty print the config to run and exit.
 
 [b]Examples[/]
-$ [i]python {sys.argv[0]} {SubCmd.launch} 1B ai2/pluto-cirrascale --launch.num_nodes=2[/]
+$ [i]python {sys.argv[0]} {SubCmd.launch} 1B Cx1 ai2/pluto-cirrascale --launch.num_nodes=2[/]
     """.strip()
 
     try:
-        script, cmd, size, cluster, overrides = (
+        script, cmd, size, run_duration, cluster, overrides = (
```

Comment:
Is there a reason to make this a required parameter and not just part of the config, with a default like 2xC?

Reply:
Well, we usually run them as a cell in the grid of {model_sizes} x {chinchilla multipliers}, so it's convenient.